### ISSUE TITLE
Biomes O'Plenty compatibility

### ISSUE THREAD

**Issue by Zarepheth** [TTFTCUTS/ATG#5]
ATG works fine when BOP is installed, but I rarely see BOP biomes, despite lots of exploration. So far, I've seen Woodland, Shrubland, and Gravel Beach biomes from BOP - otherwise, I've only seen vanilla biomes.

**Comment by GirafiStudios** [comment]
ATG does currently not have support for any modded biomes. The biomes mentioned is added by ATG. 
TTFTCUTS is currently working on support for modded biomes though!

**Comment by TTFTCUTS** [comment]
If you look at the recent commits, you can see that I'm in the middle of working on customisation and mod support. The three biomes you mentioned are added by ATG.

**Comment by jinks** [comment]
@TTFTCUTS Do you have any rough ETA? (I.e. more like weeks or more like months...)

It's not that I wanna rush you, it's just that BoP terraingen is _really_ getting on my nerves. It can't seem to generate a world I like. :)

**Comment by TTFTCUTS** [comment]
I honestly don't know. There's a lot to do, a lot to _learn_, and between working out how to do some things and finding the motivation to do so with other things eating away at my time, I couldn't possibly say right now.

**Comment by Zarepheth** [comment]
@TTFTCUTS I think I know how you feel -- I hate estimating. I usually feel like I'm offering a time period that is too high, but if I say anything smaller, I feel like I might not meet it. Worse, no matter what my estimate turns out to be, it often takes me 1/3 the time (or less) or 3 times longer (or more)! Don't those people asking for estimates know that software development is an ART, not a science? With rare exception, nearly everything I do involves unknowns that take hours of research along with trial and error before I can finish the job.

**Comment by jinks** [comment]
@TTFTCUTS Fair enough. That tells me it will be at least long enough that I shouldn't postpone any other plans in order to wait for it. :)

(I would kick myself if I started a whole new playthrough and 2 weeks later it could have been 1000% more awesome.)
